src/demo-app/card/card-demo.html

```diff
@@ -1,58 +1,58 @@
 <div class="demo-card-container">
-  <md-card>
+  <mat-card>
     Hello
-  </md-card>
+  </mat-card>
 
-  <md-card>
-    <md-card-title-group>
-        <md-card-title>Card with title</md-card-title>
-        <md-card-subtitle>Subtitle</md-card-subtitle>
-        <img md-card-md-image>
-    </md-card-title-group>
-  </md-card>
+  <mat-card>
+    <mat-card-title-group>
+        <mat-card-title>Card with title</mat-card-title>
+        <mat-card-subtitle>Subtitle</mat-card-subtitle>
+        <img mat-card-mat-image>
```

Comment:
This should be `<img mat-card-md-image>`. The second `md` here stands for "medium"